servers/quarkus-cli/build.gradle.kts

```diff
@@ -107,6 +110,41 @@ tasks.withType<QuarkusBuild>().configureEach {
   }
 }
 
+val prepareJacocoReport by
+  tasks.registering {
+    doFirst {
+      // Must delete the Jacoco data file before running tests, because
+      // quarkus.jacoco.reuse-data-file=true in application.properties.
+      file("${project.buildDir}/jacoco-quarkus.exec").delete()
+      var reportDir = file("${project.buildDir}/jacoco-report")
+      delete { delete(reportDir) }
+      reportDir.mkdirs()
+    }
+  }
+
+val jacocoReport by
```

Comment:
just wondering if it is too much trouble to aggregate all jacoco reports in our build and produce one overall _local_ report (may be in a separate PR) :thinking:

Reply:
We have that already via `codeCoverageReport`, right?

Reply:
TIL :)